docs/electric-api.yaml

```
@@ -189,4 +292,11 @@ paths:
             type: string
           description:
             Optional, deletes the current shape if it matches the shape_id.
-            If not provided, deletes the current shape.
\ No newline at end of file
+            If not provided, deletes the current shape.
+      responses:
+        '202':
+          description: No content. The shape was deleted.
```

To be slightly more honest in a generic case:
```suggestion
          description: Accepted. The shape ID is immediately invalidated, and the storage will be cleaned up eventually.
```